integration/modeshape-jbossas-integration-tests/pom.xml

```diff
@@ -231,6 +241,20 @@
                             </artifactItems>
                         </configuration>
                     </execution>
+                    
+                    <execution>
+                        <id>copy-dependencies</id>
+                        <phase>process-test-classes</phase>
+                        <goals>
+                            <goal>copy-dependencies</goal>
+                        </goals>                        
+                        <configuration>
```

Comment:
shouldn't this be configured to only copy the `org.elasticsearch` artifact-related dependencies ? In other words, don't copy all the module dependencies. Also, the generic `libs` name can then be changed to smth like `elasticsearchLibs` or something similar